packages/flutter/test/material/outline_button_test.dart

```diff
@@ -26,9 +27,12 @@ void main() {
     }
 
     await tester.pumpWidget(
-      buildFrame(() { pressedCount += 1; }),
+      buildFrame(() {
+        pressedCount += 1;
+      }),
     );
-    expect(tester.widget<OutlineButton>(find.byType(OutlineButton)).enabled, true);
+    expect(
+        tester.widget<OutlineButton>(find.byType(OutlineButton)).enabled, true);
```

Comment:
please don't indent like this; see style guide

Reply:
done